Amazon.IonObjectMapper.Test/TestObjects.cs

```diff
@@ -483,4 +483,351 @@ public static string PrettyString(IDictionary<string, int> dictionary)
+            this.ProtectedProperty = ProtectedProperty;
+            this.ProtectedInternalProperty = ProtectedInternalProperty;
+            this.InternalProperty = InternalProperty;
+            this.PrivateProperty = PrivateProperty;
+            this.PrivateProtectedProperty = PrivateProtectedProperty;
+        }
+
+        public string PublicProperty
+        {
+            get;
+            set;
+        }
+
+        string DefaultProperty
+        {
+            get;
+            set;
+        }
+
+        protected string ProtectedProperty
+        {
+            get;
+            set;
+        }
+
+        protected internal string ProtectedInternalProperty
+        {
+            get;
+            set;
+        }
+
+        internal string InternalProperty
+        {
+            get;
+            set;
+        }
+
+        private string PrivateProperty
+        {
+            get;
+            set;
+        }
+
+        private protected string PrivateProtectedProperty
+        {
+            get;
+            set;
+        }
+
+        public string GetPrivateProtectedProperty()
+        {
+            return $"{PrivateProtectedProperty}";
+        }
+
+        public string GetPrivateProperty()
+        {
+            return $"{PrivateProperty}";
+        }
+
+        public string GetProtectedProperty()
+        {
+            return $"{ProtectedProperty}";
+        }
+
+        public string GetDefaultProperty()
+        {
+            return $"{DefaultProperty}";
+        }
+    }
+
+    public class ClassWithReadonlyProperties
+    {
+        public ClassWithReadonlyProperties() { }
+
+        public ClassWithReadonlyProperties(string PublicProperty, string DefaultProperty, string ProtectedProperty, string ProtectedInternalProperty,
+            string InternalProperty, string PrivateProperty, string PrivateProtectedProperty)
+        {
+            this.PublicProperty = PublicProperty;
+            this.DefaultProperty = DefaultProperty;
+            this.ProtectedProperty = ProtectedProperty;
+            this.ProtectedInternalProperty = ProtectedInternalProperty;
+            this.InternalProperty = InternalProperty;
+            this.PrivateProperty = PrivateProperty;
+            this.PrivateProtectedProperty = PrivateProtectedProperty;
+        }
+
+        public string PublicProperty
+        {
+            get;
+        }
+
+        string DefaultProperty
+        {
+            get;
+        }
+
+        protected string ProtectedProperty
+        {
+            get;
+        }
+
+        protected internal string ProtectedInternalProperty
+        {
+            get;
+        }
+
+        internal string InternalProperty
+        {
+            get;
+        }
+
+        private string PrivateProperty
+        {
+            get;
+        }
+
+        private protected string PrivateProtectedProperty
+        {
+            get;
+        }
+
+        public string GetPrivateProtectedProperty()
```

Comment:
I presume the reason for these functions is because you can't reference private stuff in the tests.
Why not just return the property as is (rather than the string representation)?

Reply:
You are correct, I copied over the `ToString()` function from another test object and forgot about removing the string representation parsing. Updated to return property.